_vendor/github.com/docker/scout-cli/docs/docker_scout_cache.yaml

```diff
@@ -0,0 +1,28 @@
+command: docker scout cache
+short: Manage Docker Scout cache and temporary files
+long: Manage Docker Scout cache and temporary files
+pname: docker scout
+plink: docker_scout.yaml
+cname:
+    - docker scout cache df
+    - docker scout cache prune
+clink:
+    - docker_scout_cache_df.yaml
+    - docker_scout_cache_prune.yaml
+inherited_options:
+    - option: debug
+      value_type: bool
+      default_value: "false"
+      description: Debug messages
+      deprecated: false
+      hidden: true
+      experimental: false
+      experimentalcli: false
+      kubernetes: false
+      swarm: false
+deprecated: false
+experimental: false
+experimentalcli: false
+kubernetes: false
+swarm: false
+
```

Comment:
Seems like the cache topic is missing usage or something. Compared to another parent command like https://deploy-preview-18073--docsdocker.netlify.app/engine/reference/commandline/network/